lib/member_records.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require 'scraped'
+require_relative 'member_record'
+
+class MemberRecords < Scraped::JSON
+  field :members do
+    all_members
+  end
+
+  field :members_of_the_lower_house do
+    lower_house_members.map do |mem|
+      fragment mem => MemberRecord
+    end
+  end
+
+  field :next do
+    json[:next]
+  end
+
+  private
+
+  def all_members
```

Comment:
This (and probably the `members_of_the_lower_house` method too) appear to have a slightly misleading name. The implication is that a caller will actually get _all_ members, but as far as I can tell, they're only getting a subset, due to pagination, and need to call next to get the rest.